internal/testing/types/bool.go

```diff
@@ -4,13 +4,15 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/hashicorp/terraform-plugin-go/tftypes"
```

Comment:
Do you happen to have `goimports` enabled in your editor on save? It should revert this to the way it was.

Reply:
I have checked, and both Goland and `goimports` do the same thing: they seem to be putting "standard", then "3rd party", then "local" imports, separated by a space.

I thought that was the "Go way" to do imports, no?

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#imports tends to be my jam. Not sure why certain things would default to splitting out the "local" imports group versus not.